include/mbgl/util/image.hpp

```diff
@@ -26,6 +28,18 @@ class Image {
           height(h),
           data(std::move(data_)) {}
 
+    Image(Image&& o)
+        : width(o.width),
+          height(o.height),
+          data(std::move(o.data)) {}
+
+    Image& operator=(Image&& o) {
```

Comment:
That was using the default move operator. Objects with non-copyable members such as `std::unique_ptr` don't have a default copy operator.

I don't think any of the changes to image.hpp here were necessary. What was the rationale here?